lib/util/ssr.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+'use strict';
+
+module.exports.isSSREscape = function isSSREscape(node) {
+    return (
+        node.type === 'IfStatement' &&
+        node.test.type === 'BinaryExpression' &&
+        node.test.left.type === 'UnaryExpression' &&
+        node.test.left.operator === 'typeof' &&
+        node.test.left.argument.type === 'Identifier' &&
+        (node.test.left.argument.name === 'document' || node.test.left.argument.name === 'window')
```

Comment:
This is also something we should parameterize. I would expect LWR to come with a more elegant option than `type 'window'` checks.

Reply:
I think if we land `imports.meta.env`, we can change this to check for that.

Reply:
Sound good to me. Could you add a comment here, to make sure we track this in the future?